inspire_disambiguation/api.py

```diff
@@ -78,32 +81,42 @@ def train_and_save_distance_model(
             will be saved.
         sampled_pairs_size (int): Number of pairs to be generated for the training.
             Note:
-                Must be multiple of 12.
+                Must be multiple of 4.
     """
     LOGGER.info("Pulling training data from ES")
     start_time = datetime.now()
     curated_signatures = get_signatures(only_curated=True)
-    input_clusters = get_input_clusters(curated_signatures)
+    train_signatures_dict, test_signatures_dict = \
+        train_validation_split(curated_signatures, train_to_validation_split_fraction)
+    train_signatures_list = [train_signatures_dict[signature] for signature in train_signatures_dict.keys()]
+    test_signatures_list = [test_signatures_dict[signature] for signature in test_signatures_dict.keys()]
```

Comment:
I think you can just do
```suggestion
    train_signatures = train_signatures_dict.values()
    test_signatures = test_signatures_dict.values()
```
and wrap this in a list if you really need a list (but I doubt you actually do).